erpnext/stock/doctype/landed_cost_voucher/landed_cost_voucher.py

```
@@ -427,7 +429,7 @@ def get_pr_items(purchase_receipt):
 
 @frappe.whitelist()
 @frappe.validate_and_sanitize_search_inputs
-def get_vendor_invoices(doctype, txt, searchfield, start, page_len, filters):
+def get_vendor_invoices(doctype: str, txt: str, searchfield: Any, start: int, page_len: int, filters: dict):
```

Ruff (0.15.1)

[warning] Unused function argument: `searchfield`

(ARG001)